crates/vm/levm/src/hooks/default_hook.rs

```diff
@@ -29,7 +29,13 @@ impl Hook for DefaultHook {
     ///   See 'docs' for more information about validations.
     fn prepare_execution(&self, vm: &mut VM<'_>) -> Result<(), VMError> {
         let sender_address = vm.env.origin;
-        let sender_account = vm.db.get_account(sender_address)?;
+        let sender_balance;
+        let sender_nonce;
+        {
+            let sender_account = vm.db.get_account(sender_address)?;
+            sender_balance = sender_account.info.balance;
+            sender_nonce = sender_account.info.nonce;
+        }
```

Comment:
```suggestion
       let (sender_balance, sender_nonce) = {
           let sender_account = vm.db.get_account(sender_address)?;
           (sender_account.info.balance, sender_account.info.nonce)
       };
```
Do you think this looks a little bit prettier?

Reply:
I hadn't thought of that! But it seems like a good idea.